### DISCUSSION TITLE
Adding microformats to templates

### DISCUSSION THREAD

**Discussion by dltj** [mmistakes/minimal-mistakes#3017] [Ideas]
I'm looking at connecting [my blog](https://dltj.org/) with [WebmentionIO](https://webmention.io/dashboard) and [Bridgy](https://brid.gy/), but that requires adding Microformats (as outlined on the https://indiewebify.me/ site).  I noted back in https://github.com/mmistakes/minimal-mistakes/issues/153#issuecomment-114156988 from 2015 that you got frustrated with Microformats and ripped them out.  Would you be open to a pull request that adds them back in?

**Comment by mmistakes** [comment]
Hello. Yes I'm open to a pull request that adds Microformats to the theme.
As long as it's done in a way that is transparent to the user and doesn't require adding all sorts of unecessary markup (think this was more of an issue with schema markup back in the day than Microformats) I'm cool with it.

Feel free to open a pull request and ping me for feedback or a review.

**Reply by dltj** [reply]
Great!  I created https://github.com/mmistakes/minimal-mistakes/pull/3052 with the microformats work.  You can see it in action here on a post where I have added Webmention.IO hooks (using https://github.com/aarongustafson/jekyll-webmention_io as the glue between Jekyll and Webmention.IO):

https://dltj.org/article/digital-preservation-software/

Let me know what you think.
